specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -71,6 +75,23 @@
         }
       }
     },
+    "DatasetSchemaDataElement": {
```

Comment:
Where is this model referenced?

Reply:
It will be used for the newly added property "schema". Though we can't just refer "DatasetSchemaDataElement" there due to the "schema" property can take other types (i.e. string, expression object...) which eventually will convert to DatasetSchemaDataElement. This property actually just follows the same usage as "DatasetDataElement"